tools/example-templates/additional-files/Attr2Attribute.cs

```diff
@@ -0,0 +1,14 @@
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+public class Attr2Attribute : Attribute
```

Comment:
Why are various parts here commented out?

Reply:
removed

Reply:
They don't appear to be, at the moment. Is there a commit missing?